libvips/foreign/svgload.c

```diff
@@ -1115,7 +1116,7 @@ vips_foreign_load_svg_buffer_init(VipsForeignLoadSvgBuffer *buffer)
  * During the CSS cascade, the specified stylesheet will be applied with a
  * User Origin. This feature requires librsvg 2.48.0 or later.
  *
- * Setting @high_bitdepth TRUE enables 128-bit scRGB output."
+ * Set @high_bitdepth to enable 128-bit scRGB output (32-bit per channel).
```

Comment:
Maybe we should say "request"? The flag doesn't guarantee float output and you'll get a warning and uchar if it's not supported.

Perhaps `svgload` should fail if float rendering is not available? I can't decide :( 

OTOH, pretty soon all librsvgs will support this, so failure won't be possible and it won't matter.

Reply:
... let's leave this alone.